docs/concepts/advanced/rendering_js_css.md

```diff
@@ -220,3 +220,203 @@ with [`Component.render()`](#TODO) and inserting them into larger structures.
     - Directly passing rendered HTML to [`render_dependencies()`](#TODO)
 3. If you pre-render one component to pass it into another, the pre-rendered component must be rendered with
    [`render_dependencies=False`](#TODO).
+
+## Dependencies strategies
+
+The rendered HTML may be used in different contexts (browser, email, etc).
+If your components use JS and CSS scripts, you may need to handle them differently.
+
+[`render()`](../../../reference/api/#django_components.Component.render) and [`render_to_response()`](../../../reference/api/#django_components.Component.render_to_response)
+accept a `deps_strategy` parameter, which controls where and how the JS / CSS are inserted into the HTML.
+
+The `deps_strategy` parameter is set at the root of a component render tree, which is why it is not available for
+the [`{% component %}`](../../../reference/template_tags#component) tag.
+
+!!! info
+
+    The `deps_strategy` parameter is ultimately passed to [`render_dependencies()`](../../../reference/api/#django_components.render_dependencies).
+
+There are five dependencies strategies:
+
+- [`document`](../../advanced/rendering_js_css#document) (default)
+    - Smartly inserts JS / CSS into placeholders or into `<head>` and `<body>` tags.
+    - Inserts extra script to allow `fragment` strategy to work.
+    - Assumes the HTML will be rendered in a JS-enabled browser.
+- [`fragment`](../../advanced/rendering_js_css#fragment)
+    - A lightweight HTML fragment to be inserted into a document with AJAX.
+    - No JS / CSS included.
+- [`simple`](../../advanced/rendering_js_css#simple)
+    - Smartly insert JS / CSS into placeholders or into `<head>` and `<body>` tags.
+    - No extra script loaded.
+- [`prepend`](../../advanced/rendering_js_css#prepend)
+    - Insert JS / CSS before the rendered HTML.
+    - No extra script loaded.
+- [`append`](../../advanced/rendering_js_css#append)
+    - Insert JS / CSS after the rendered HTML.
+    - No extra script loaded.
+
+### `document`
+
+`deps_strategy="document"` is the default. Use this if you are rendering a whole page, or if no other option suits better.
```

Comment:
I wonder if this should be a setting too? Easier to disable the dependency manager once than for every component.

Reply:
Sounds good, make a follow up ticket https://github.com/django-components/django-components/issues/1159